user-interface/src/main/java/life/qbic/datamanager/views/projects/create/CollaboratorsLayout.java

```diff
@@ -118,6 +118,18 @@ public ProjectCollaborators getProjectCollaborators() {
     return projectCollaborators;
   }
 
+  public void setKnownContacts(List<Contact> knownContacts) {
+    principalInvestigatorField.setItems(knownContacts);
+    responsiblePersonField.setItems(knownContacts);
+    projectManagerField.setItems(knownContacts);
+  }
+
+  public void hideContactBox() {
+    principalInvestigatorField.hideContactBox();
+    responsiblePersonField.hideContactBox();
+    projectManagerField.hideContactBox();
+  }
```

Comment:
Even fancier would be a method like formlayout.showContactBox(knownContacs.isEmpty()) that sets the visibility dependent on the boolean provided

Reply:
I think this is unnecessary, at least at the moment, because there is no possibility that the visibility will be set to true.